src/stylesheets/app.scss

```diff
@@ -1,32 +1,77 @@
 // Global styles
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;0,700;1,100&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Average+Sans&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Hind+Madurai&display=swap");
 
 * {
   margin: 0;
   padding: 0;
+  overflow: hidden;
 }
 
-*, *::after, *::before {
+*,
+*::after,
+*::before {
   box-sizing: border-box;
 }
 
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family: "Hind Madurai", sans-serif;
+  font-size: 36px;
```

Comment:
I think this should be changed to 16px, as this is the standard font-size for websites. We can scale up element font-sizes by using relative sizes, e.g. 2em.
If we set `font-size: 36px` here then it'll carry through to every other element, which will make it hard to put in regular sized text